### ISSUE TITLE
Unable to view last requests in dynamic filtering panel due to no scrolling

### ISSUE THREAD

**Issue by fe2-Nyxar** [uBlockOrigin/uBlock-issues#3504]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

I'm using uBlock Origin in Advanced Mode, which allows me to monitor and block individual requests from websites dynamically. However, I’ve encountered a usability issue:
    On some websites with many requests to third-party domains, the dynamic filtering panel doesn't show the last few requests.
    There’s no way to scroll to view or manage these hidden requests.
    This makes it difficult to block or allow specific requests when the list is long, especially on sites that make a high volume of requests.

### A specific URL where the issue occurs.

```yaml
it happens on every site with many third-party requests, here's an example: https://suno.com/song/48b1ea72-7ccd-446d-a3a4-54c699915b21
```

### Steps to Reproduce

Enable Advanced Mode in uBO.
Visit a website that makes many third-party requests (e.g., ad-heavy or analytics-heavy sites).
Open the Dynamic Filtering Panel.
Notice that the last few requests are inaccessible even if you scroll

### Expected behavior

The dynamic filtering panel should include a scrollbar or some mechanism to access all requests, regardless of how many are made by the site. For accessing all requests, regardless of how many the site makes

### Actual behavior

The panel displays only a fixed number of requests, with no ability to scroll or view the remaining ones. The last few requests become inaccessible, and it is impossible to view 

https://github.com/user-attachments/assets/af41cc68-5ec0-446a-9515-54e30adc8834

### uBO version

 Version 1.61.2

### Browser name and version

Mozilla Firefox 133.0

### Operating System and version

Nixos 24.11

**Comment by gwarser** [comment]
I see the scrollbar on your video. The thin transparent line on the left border of the popup. There is no more top level domains on this page. Subdomains are hidden - click the `+ all` in first row or click the rows with dots before the top domain name to see them.

Hmm, do you mean it has rubber-band effect? If you scroll to the bottom it scrolls back by itself? Your video looks small is this your monitor resolution?

**Comment by gwarser** [comment]
OK, I see the issue when I set the screen resolution to 1280x720, the `statusz.` subdomain is not visible in the popup, but it is when opened in new window:

![Screenshot1](https://github.com/user-attachments/assets/c0168c53-a064-42f8-a092-8cdb0763b2c6)
![Screenshot1](https://github.com/user-attachments/assets/471d6e08-8c5a-41ad-ad96-96960def6e95)

**Comment by gorhill** [comment]
I get `suno.ai` as the last item in the list, there is nothing to see beyond that entry. That entry is properly shown in your video.

**Comment by gwarser** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/3014

**Comment by fe2-Nyxar** [comment]
> I get `suno.ai` as the last item in the list, there is nothing to see beyond that entry. That entry is properly shown in your video.

Yes, but I can't block it or unblock it because it's at the end of the list

**Comment by fe2-Nyxar** [comment]
> Duplicate of #3014

yes that's the exact issue, he did a better job explaining it, is it an issue at my end or what?

**Comment by gorhill** [comment]
The browser is giving uBO a container to render its popup panel and uBO doesn't know it's off screen, only the browser knows. The fix has to be browser-side, it needs to ensure the container in which uBO is told to render to not extend beyond the screen.

Meanwhile a workaround is to open the panel in its own tab, by pressing <kbd>Ctrl</kbd>+<kbd>Shift</kbd> and clicking the `all` cell.

**Comment by u-RraaLL** [comment]
> Meanwhile a workaround is to open the panel in its own tab, by pressing <kbd>Ctrl</kbd>+<kbd>Shift</kbd> and clicking the `all` cell.

Or by scroll-clicking uBO's shield.

**Comment by fe2-Nyxar** [comment]
> > Meanwhile a workaround is to open the panel in its own tab, by pressing Ctrl+Shift and clicking the `all` cell.
> 
> Or by scroll-clicking uBO's shield.

how is that?

**Comment by fe2-Nyxar** [comment]
> The browser is giving uBO a container to render its popup panel and uBO doesn't know it's off screen, only the browser knows. The fix has to be browser-side, it needs to ensure the container in which uBO is told to render to not extend beyond the screen.
> 
> Meanwhile a workaround is to open the panel in its own tab, by pressing Ctrl+Shift and clicking the `all` cell.

Thank you man, well I can work with this for the moment :)

**Comment by gorhill** [comment]
> how is that?

Click the middle button if you have one -- I had forgotten about this.

**Comment by fe2-Nyxar** [comment]
> > how is that?
> 
> Click uBO's toolbar icon using your mouse's middle button if you have one -- I had forgotten about this.

thank you it works :+1:

**Comment by gwarser** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1943386

**Comment by fe2-Nyxar** [comment]
> https://bugzilla.mozilla.org/show_bug.cgi?id=1943386

perfect 👍 , let's hope they'll fix it
